style.css

```diff
@@ -68,43 +73,85 @@ ul li {
   cursor: pointer;
 }
 
-ul li.complete {
+li.complete {
   color: #555;
   text-decoration: line-through;
 }
 
-ul li deletebutton {
-  position: absolute;
-  right: 0;
-  top: 3px;
-  width: 40px;
-  height: 40px;
+.btn-edit,
+.btn-delete {
+  padding: 8px 12px;
+  top: 10px;
   font-size: 12px;
   font-weight: 700;
-  color: red;
-  line-height: 40px;
+  color: white;
   text-align: center;
-  border-radius: 70%;
+  outline: none;
+  border: none;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0;
+  line-height: 10px;
+}
+
+.btn-delete {
+  position: absolute;
+  right: 0;
+  background-color: red;
 }
 
-ul li deletebutton:hover {
+.btn-delete:hover {
   background: rgb(225, 223, 223);
+  color: red;
 }
 
-ul li editbutton {
+.btn-edit {
   position: absolute;
-  right: 50px;
-  top: 3px;
-  width: 40px;
-  height: 40px;
-  font-size: 12px;
-  font-weight: 700;
-  color: black;
-  line-height: 40px;
-  text-align: center;
-  border-radius: 70%;
+  right: 65px;
+  background-color: #26ca10;
 }
 
-ul li editbutton:hover {
+.btn-edit:hover {
   background: rgb(225, 223, 223);
+  color: #26ca10;
```

Comment:
Try not to mix different approaches, if you decided to use RGB - use it everywhere, if you decided to use hex - use it, but try not to mix them. In addition, you can also put all the colors into separate variables, this will help for example with creating themes (dark/light/ etc).